### ISSUE TITLE
Feature request: X-mouse copy

### ISSUE THREAD

**Issue by jamiesonbecker** [stefansundin/altdrag#22]
Selecting text (only) copies into a separate buffer (not the control-C/control-V buffer), middle click in a text area or console window pastes from that buffer. (Putty does support some version of this but uses the Control-C/Control-V buffer and the non-standard right mouse button.)

**Comment by stefansundin** [comment]
See True X-Mouse for this behavior: http://fy.chalmers.se/~appro/nt/TXMouse/

**Comment by jamiesonbecker** [comment]
Right, thx! It doesn't work quite right but I guess it's the best there 
is at the moment. :)
